Source/WebKit2/WebProcess/WebPage/wpe/WebPageWPE.cpp

```diff
@@ -39,7 +39,26 @@ namespace WebKit {
 
 void WebPage::platformInitialize()
 {
+#if HAVE(ACCESSIBILITY)
+    // Create the accessible object (the plug) that will serve as the
+    // entry point to the Web process, and send a message to the UI
+    // process to connect the two worlds through the accessibility
+    // object there specifically placed for that purpose (the socket).
+    m_accessibilityObject = adoptGRef(webPageAccessibilityObjectNew(this));
+    GUniquePtr<gchar> plugID(atk_plug_get_id(ATK_PLUG(m_accessibilityObject.get())));
+    // send(Messages::WebPageProxy::BindAccessibilityTree(String(plugID.get())));
+#endif
+}
+
+#if HAVE(ACCESSIBILITY)
+void WebPage::updateAccessibilityTree()
```

Comment:
This function is only called from WebFrameLoaderClient::dispatchDidClearWindowObjectInWorld, where it's guarded by `#if HAVE(ACCESSIBILITY) && (PLATFORM(GTK) || PLATFORM(EFL))`. So the code you added here will never be called.

Reply:
Hi @mcatanzaro, I just realized that I missed that file while preparing the patch. Thank you for the review, will add a PR with the change.